cstar/roms/component.py

```diff
@@ -471,3 +442,76 @@ def post_run(self) -> None:
                     cwd=out_path,
                     shell=True,
                 )
+
+
+class ROMSDiscretization(Discretization):
+    """
+    An implementation of the Discretization class for ROMS.
+
+
+    Additional attributes:
+    ----------------------
+    n_procs_x: int
+        The number of parallel processors over which to subdivide the x axis of the domain.
+    n_procs_y: int
+        The number of parallel processors over which to subdivide the y axis of the domain.
+
+
+    Properties:
+    -----------
+    n_procs_tot: int
+        The value of n_procs_x * n_procs_y
+
+    """
```

Comment:
I expect the docs formatters to complain about these extra blank lines
```suggestion
    """
    An implementation of the Discretization class for ROMS.

    Additional attributes:
    ----------------------
    n_procs_x: int
        The number of parallel processors over which to subdivide the x axis of the domain.
    n_procs_y: int
        The number of parallel processors over which to subdivide the y axis of the domain.

    Properties:
    -----------
    n_procs_tot: int
        The value of n_procs_x * n_procs_y
    """
```

Reply:
I'll issue a separate PR for docstring cleanup seeing as ruff has found hundreds of errors now I've added a check